content/applications/finance/fiscal_localizations/philippines.rst

```diff
@@ -0,0 +1,120 @@
+===========
+Philippines
+===========
+
+Introduction
+============
+
+The **l10n_ph** module adds :doc:`accounting localization <../fiscal_localizations>` features for
+the Philippines such as a chart of accounts, taxes, and the BIR 2307 report. These provide a base
+template to get started with using Philippine accounting.
+
+Configuration
+=============
+
+Module installation
+-------------------
+
+:ref:`Install <general/install>` the **l10n_ph** module to get all the features of the Philippine
+localization:
+
+.. image:: philippines/philippines-module.png
+   :alt: Philippines fiscal localisation module.
+
+.. note::
+   - When creating a new database and `Philippines` is selected as a country, the fiscal
+     localization module :guilabel:`Philippines - Accounting` is automatically installed.
+   - If installing the module in an existing company, the Chart of Accounts and Taxes will *not* be
+     replaced if there are already posted journal entries.
+   - The BIR 2307 report will be installed but the withholding taxes may need to be manually
+     created.
+
+Chart of accounts & taxes
+-------------------------
+
+A minimum configuration default chart of accounts is installed and the following types of taxes are
+installed and also linked to the relevant account:
+
+- VAT 12%
+- VAT Exempt
+- Withholding taxes
+
+For the withholding taxes, there is an additional :guilabel:`Philippines ATC` field under the
+:guilabel:`Philippines` tab.
+
+.. image:: philippines/philippines-atc-code.png
+   :alt: Philippines ATC code field set on taxes.
+
+This is the ATC code for that tax when it is exported in the BIR 2307 file and will need to be
+filled in manually if the taxes are manually created.
+
+Contacts
+--------
+
+If a contact record is set as a :guilabel:`Company` and the :guilabel:`Country` is `Philippines`,
+fill in the :guilabel:`Tax ID` field with the `Taxpayer Identification Number (TIN)`.
+
+.. image:: philippines/philippines-contact-company.png
+   :alt: Company type contact with Tax ID field.
+
+If a contact record is set as a :guilabel:`Individual`, the :guilabel:`Country` is `Philippines` and
+it does not belong to any company, fill in the :guilabel:`Tax ID` field with the `Taxpayer
+Identification Number (TIN)`.
+There are additional fields for:
+
+- :guilabel:`First Name`
+- :guilabel:`Middle Name`
+- :guilabel:`Last Name`
```

Comment:
@xpl-odoo I like your shortened version. What's your reasoning for keeping the second screenshot? Looking at it now, I feel like both the company and individual screenshots could be removed.

Reply:
Just because there are fields that are specific to this localization.
You can remove both, in my opinion. The screenshots repeat what is being said, making them not so useful.

Reply:
@dade-odoo you mentioned you liked my suggestion and didn't think the screenshots were useful here. Yet this wasn't changed at all? (I mean, I'm ok with that, but you may have forgotten to change this part?)

Reply:
Good catch :) I decided to keep the single screenshot since it does show fields specific to this localization.